modules/ossm-threescale-webassembly-module-mapping-rule-object.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-mapping-rule-object_{context}"]
+= The 3scale WebAssembly module `mapping_rule` object
+
+The `mapping_rule` object is part of an array in the `mapping_rules` object.
+
+The `mapping_rule` object fields specify:
+
+* The _HTTP request method_ to match.
+* A pattern to match the path against.
+* The 3scale methods to report along with the amount to report. The order in which you specify the fields determines the evaluation order.
+
+The following table outlines the fields in the `mapping_rule` object:
+|===
+|Name |Description |Required
+
+|`method`
+|Specifies a string representing an HTTP request method, also known as verb. Values accepted match the any one of the accepted HTTP method names, case-insensitive. A special value of any matches any method.
+|Yes
+
+|`pattern`
+|The pattern to match the HTTP request's URI path component. This pattern follows the same syntax as documented by 3scale. It allows wildcards (use of the asterisk (*) character) using any sequence of characters between braces such as {this}.
+|Yes
+
+|`usages`
+a|This is a list of `usage` objects. When the rule matches, all methods with their `deltas` are added to the list of methods sent to 3scale for authorization and reporting.
```

Comment:
s/This is a list/A list of/

To match the style of the other Description entries in the table a little closer.